SDWebImage/SDWebImageDownloaderOperation.h

```diff
@@ -36,16 +36,17 @@ FOUNDATION_EXPORT NSString * _Nonnull const SDWebImageDownloadFinishNotification
 - (nullable id)addHandlersForProgress:(nullable SDWebImageDownloaderProgressBlock)progressBlock
                             completed:(nullable SDWebImageDownloaderCompletedBlock)completedBlock;
 
-- (nullable NSURLCredential *)credential;
```

Comment:
This `NSURLCredential` don't need to be required method in protocol. The requried method should be the important method for basic network request. So it's better to move the optional method instead